programs/axelar-solana-gateway/src/instructions.rs

```diff
@@ -226,16 +225,22 @@ pub enum GatewayInstruction {
     TransferOperatorship,
 }
 
+/// Represents an initial verifier set with its hash and PDA
+#[derive(Debug, Clone, PartialEq, Eq, BorshSerialize, BorshDeserialize)]
+pub struct InitialVerifierSet {
+    /// The hash of the verifier set
+    pub hash: VerifierSetHash,
+    /// The PDA for the verifier set tracker
+    pub pda: Pubkey,
+}
+
```

Comment:
I decided to use a dedicated struct here to be super clear that this should only be used to initialize the Gateway.